stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -121,14 +131,21 @@ extension TensorHandle : TensorSendableReceivable {
       dumpTensorContent(self.cTensorHandle, Scalar.self)
     }
     let status = TF_NewStatus()
-    let cTensor = TFE_TensorHandleResolve(self.cTensorHandle, status)
-    checkOk(status)
-    TF_EnqueueNamedTensor(
-      computation.cSession, Int32(tensorId), cTensor, status)
+    internalConsistencyCheck(status != nil)
+    if _RuntimeConfig.usesTFEagerAPI {
+      let context = _ExecutionContext.global
+      TFE_EnqueueNamedTensorFromCtx(
+        context.eagerCtx, Int32(tensorId), self.cTensorHandle, status)
```

Comment:
`tensorId` -> `tensorID`

Reply:
Changed, but this is existing code, and I feel `tensorId` would also be fine.

Reply:
In Swift readability reviews we tend to prefer "ID", but it's undocumented in Google's swift style guide. The way I reason about it is as follows:
- "Id" is a shortened version of "identifier".
- "ID" is fine and does not need to be considered a shortened name, because in English "ID" is a word.